www/src/theme.js

```diff
@@ -257,7 +257,7 @@ export const DEFAULT_THEME = {
       },
     },
     font: {
-      family: 'Monument',
+      family: 'Inter',
```

Comment:
It looks like in most of the new designs Inter font is being used. Monument is still used in buttons etc. but this comes from the design system components.